website/src/content/api-reference/commercetools-frontend-actions-global.mdx

```diff
@@ -0,0 +1,240 @@
+---
+title: "@/actions-global"
+---
+
+<Subtitle>
+
+Global Redux actions and hooks for handling user notifications.
+
+</Subtitle>
+
+# Installation
+
+```
+yarn add @commercetools-frontend/actions-global
+
+# or
+
+npm --save install @commercetools-frontend/actions-global
+```
+
+Additionally install the peer dependencies (if not present)
+
+```
+yarn add react redux react-redux
+
+# or
+
+npm --save install react redux react-redux
+```
+
+# Notification system
+
+Custom Applications come with a built-in notification system built on top of [Redux](https://redux.js.org).
+
+The `@commercetools-frontend/actions-global` package provides Redux action creators and React hooks to facilitate dispatching notifications within Custom Applications.
+
+# Action creators
```

Comment:
Since we recommend using React hooks, perhaps it would help to have its docs before the actions creators but I know it might not be feasible because hooks descriptions depend on actions creators ones.